.github/workflows/format.yml

```
@@ -29,7 +29,7 @@ jobs:
           format_files: true
         # Fail on next step instead
         continue-on-error: ${{ github.event_name == 'pull_request' }}
-      - uses: reviewdog/action-suggester@v1.23
+      - uses: reviewdog/action-suggester@v1.24
```

```suggestion
      - uses: reviewdog/action-suggester@v1
```

Lets just accept arbitrary 1.x versions again, the bug seems fixed.